fe/fe-core/src/main/java/com/starrocks/http/rest/LoadAction.java

```diff
@@ -94,18 +97,25 @@ public void executeWithoutPassword(BaseRequest request, BaseResponse response) t
             checkTblAuth(ConnectContext.get().getCurrentUserIdentity(), dbName, tableName, PrivPredicate.LOAD);
         }
 
-        // Choose a backend sequentially.
-        List<Long> backendIds = GlobalStateMgr.getCurrentSystemInfo().seqChooseBackendIds(1, true, false);
-        if (CollectionUtils.isEmpty(backendIds)) {
-            throw new DdlException("No backend alive.");
+        // Choose a backend sequentially, or choose a cn in shared_data mode
+        List<Long> nodeIds = new ArrayList<>();
```

Comment:
this `new ArrayList<>();` is not necessary, since the `nodeIds` will be assigned in either case.